admin/formprocess.php

```diff
@@ -72,42 +64,6 @@
       <div class="col-sm-12">
         <strong><em><?php echo $message; ?></em></strong>:
       </div>
-      <div class="col-sm-12 headline">
-        <h2><?php echo $series_name . " #" . $issue_number; ?></h2>
-        <a href="#">&lt; Back</a>
-      </div>
-      <div class="col-md-8">
-        <div class="issue-details">
-          <h3>Issue details</h3>
-          <?php
-            if ($details->writer) {
-              echo "<div class=\"issue-writer\">";
-              echo "Writer: " . $details->writer;
-              echo "</div>";
-            }
-
-            if ($details->artist) {
-              echo "<div class=\"issue-artist\">";
-              echo "Artist: " . $details->artist;
-              echo "</div>";
-            }
-          ?>
-          <div>
-            <strong>Issue: </strong><?php echo $issue_number; ?>
-          </div>
-          <div>
-            <strong>Story Name: </strong><?php echo $story_name; ?>
-          </div>
-          <div>
-            <strong>Published: </strong><?php echo $released_date; ?>
-          </div>
-          <a href="../comic.php?comic_id=<?php echo $comic_id; ?>">Go to record</a>
-        </div>
-        <div class="issue-description"><?php echo $plot; ?></div>
-      </div>
-      <div class="col-md-4 issue-image">
-        <img src="../images/<?php echo $cover_image_file; ?>" />
-      </div>
```

Comment:
As the page is redirected to the issue page, this content is no longer necessary and will prevent layout discrepancies.